docs/pages/faq.js

```diff
@@ -8,9 +8,9 @@ import Page from '../components/Page.js';
 
 export default function ContainerPage(): Node {
   return (
-    <Page title="Frequently Asked Questions">
-      <PageHeader name="Frequently Asked Questions" showSourceLink={false} />
-      <Card name="Gestalt Usage">
+    <Page title="Frequently asked questions">
```

Comment:
Should these remain capitalized, given the "FAQ" acronym?

Reply:
I think sentence case makes sense here - but totally open to change if we strongly believe it should be capitalize.